index.html

```diff
@@ -0,0 +1,169 @@
+          <li class="d-naver">NAVER</li>
+          <li class="d-airbnb">airbnb</li>
+          <li class="d-airoflot">Airoflot</li>
+        </ul>
+      </div>
+      <footer class="d-footer">
+        <div class="d-copy">@creative <br> Commons Kerea</div>
+        <div class="alright-reserved">
+          <p class="d-footer-content">Creative commons korea. Some rights reserved.</p>
+        </div>
+      </footer>
+    </section>
+  
+  <!-- About page section -->
+  <section id="about-course" class="about-page">
+    <div class="about-page-card">
+        <h2 class="about-page-title">
+          INPROVE YOUR LIFE THROUGH LEARNING
+        </h2>
+        <p class="about-page-content">
+          This course is designed for teachers delivering lessons at a low level
+          or equivalent, and for beginner programmers looking to start using HTML, CSS
+          and JavaScript. To take this course, you need to be familiar with basic 
+          programming structures such as loops and if statements, but you don't 
+          need any experience of HTML, CSS or JavaScript.
+        </p>
+    </div>
+    <div class="about-page-contact">
+      <p>Please contact us per Email for any further questions about Youcode online course.
+       <p class="email-address">ezraermy@gmail.com</p>
+      </p>
+    </div>
+    <div class="about-page-logo">
+      <h3 class="about-page-logo-title">
+        Youcode web-dev course logo
+        <hr>
+      </h3>
+      <p class="about-page-logo-competition">The logo of youcode web-dev site hasn't been choosen yet. Will be choosen through the logo competition from 8.June to 7.July</p>
+      <img src="images/logo2.jpg" class="about-page-logo-img" alt="">
+    </div>
+    <div class="about-page-past-summit">
+      <h3 class="about-page-past-title">See the past youcode student's success</h3>
+      <hr>
+      <p class="about-page-logo-competition">
+        Take a look at the last two youcode web-dev studen's graduation which took place in Brazil and Poland.
+      </p>
+      <div class="d-about-page-past-img">
+        <img src="images/past-img0.png" alt="Past images of the program" class="about-page-past-img">
+        <img src="images/past-img1.png" alt="Past images of the program" class="about-page-past-img">
+      </div>
+      <div class="partner">
+        <h4 class="partner-title">Partners</h4>
+        <hr>
+        <ul class="sponsors">
+          <li class="mozilla">Mozilla</li>
+          <li class="google">Goole</li>
+          <li class="naver">IBM</li>
+          <li class="airbnb">airbnb</li>
+          <li class="airoflot">IMB</li>
+        </ul>
+      </div>
+    </div>
+  </section>
+  <footer class="footer">
+    <div class="copy">@Creative Commons Kerea</div>
+    <div class="alright-reserved">
+      <p class="footer-content">Creative commons korea. Some rights reserved.</p>
+    </div>
+  </footer>
+  <footer class="desktop-footer">
+    <div class="copy">@Creative Commons Kerea</div>
```

Comment:
- [ ] Kindly make sure to have a two pages website, the home and about page, created with different HTML files

![Screenshot from 2023-02-10 22-53-52](https://user-images.githubusercontent.com/97834160/218196345-f8414e23-a12d-4996-88ef-7e6944d4bca6.png)


__ As you can see below this part is supposed to be for the about page__